modules/local/star_align_igenomes.nf

```diff
@@ -0,0 +1,70 @@
+process STAR_ALIGN_IGENOMES {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::star=2.6.1d bioconda::samtools=1.10 conda-forge::gawk=5.1.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-1fa26d1ce03c295fe2fdcf85831a92fbcbd7e8c2:59cdd445419f14abac76b31dd0d71217994cbcc9-0' :
+        'quay.io/biocontainers/mulled-v2-1fa26d1ce03c295fe2fdcf85831a92fbcbd7e8c2:59cdd445419f14abac76b31dd0d71217994cbcc9-0' }"
+
+    input:
+    tuple val(meta), path(reads)
```

Comment:
This already exists as an nf-core module

Reply:
Fixed in #77 - replaced local version with nf-core version

Reply:
I believe there is an on-going issue with the STAR version and iGenomes index. See [here](https://github.com/nf-core/rnaseq/pull/810). We wanted to be consistent with nf-core/rnaseq as most use-cases will be to take the BAM file from that workflow and process it with rnasplice for downstream analysis.